publishable/config/voyager.php

```diff
@@ -11,6 +11,7 @@
     */
 
     'user' => [
+        'model'                        => \App\User::class,
```

Comment:
Need to also update the "dummy" config: https://github.com/the-control-group/voyager/blob/1.3/publishable/config/voyager_dummy.php